src/test/scala/com/fulcrumgenomics/bam/api/SamRecordTest.scala

```diff
@@ -27,9 +27,10 @@ package com.fulcrumgenomics.bam.api
 import com.fulcrumgenomics.alignment.Cigar
 import com.fulcrumgenomics.testing.SamBuilder.{Minus, Plus}
 import com.fulcrumgenomics.testing.{SamBuilder, UnitSpec}
-import htsjdk.samtools.{SAMRecord, TextCigarCodec}
+import htsjdk.samtools.TextCigarCodec
+import org.scalatest.OptionValues
 
-class SamRecordTest extends UnitSpec {
+class SamRecordTest extends UnitSpec with OptionValues {
```

Comment:
Should we just addd `OptionValues` into `UnitSpec`?

Reply:
I am hesitant to roll to much into UnitSpec.  Lets leave as is for now.